physionet-django/events/templatetags/participation_status.py

```diff
@@ -21,4 +22,4 @@ def is_on_waiting_list(user, event):
 
 @register.filter(name='has_access_to_event_dataset')
 def has_access_to_event_dataset(user, dataset):
-    return dataset.has_access(user)
+    return has_access_to_event_dataset_func(user, dataset)
```

Comment:
Do we need to rename the function here? Not sure it is necessary, and it is inconsistent with the approach used below (when importing to `authorization/access.py`).

Reply:
I think Amit has moved on from this project, so I'll leave this to be addressed later.